examples/notebooks/Spark Configuration.ipynb

```diff
@@ -1,127 +1,138 @@
 {
- "cells": [
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "import pyspark\n",
-    "from pyspark.conf import SparkConf\n",
-    "from pyspark.sql import SparkSession\n",
-    "import pandas as pd\n",
-    "\n",
-    "CATALOG_URL = \"http://server:8080/catalog\"\n",
-    "MANAGEMENT_URL = \"http://server:8080/management\"\n",
-    "DEMO_WAREHOUSE = \"demo\"\n",
-    "SPARK_VERSION = pyspark.__version__\n",
-    "SPARK_MINOR_VERSION = '.'.join(SPARK_VERSION.split('.')[:2])\n",
-    "ICEBERG_VERSION = \"1.5.2\""
-   ]
+  "cells": [
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "import pyspark\n",
+        "from pyspark.conf import SparkConf\n",
+        "from pyspark.sql import SparkSession\n",
+        "import pandas as pd\n",
+        "\n",
+        "# edit this if you are not running TIP locally\n",
+        "CATALOG_URL = \"http://localhost:8080/catalog\"\n",
+        "\n",
+        "# warehouse needs to already exist.\n",
+        "# see USER_GUIDE.MD and STORAGE.MD for configuration examples.\n",
+        "# this example assumes AWS S3 or MinIO storage backend, \n",
```

Comment:
examples docker compose has a container initing the warehouse, should probably be noted here